Specs/Scene/ExpressionSpec.js

```diff
@@ -499,6 +501,140 @@ defineSuite([
         expect(expression.evaluate(frameState, undefined)).toEqual(0.5);
     });
 
+    it('evaluates color properties (x, y, z, w)', function() {
+        var expression = new Expression('color(\'#ffffff\').x');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('rgb(255, 255, 0).y');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('color("cyan").z');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('rgba(255, 255, 0, 0.5).w');
+        expect(expression.evaluate(frameState, undefined)).toEqual(0.5);
+    });
+
+    it('evaluates color properties ([0], [1], [2]. [3])', function() {
+        var expression = new Expression('color(\'#ffffff\')[0]');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('rgb(255, 255, 0)[1]');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('color("cyan")[2]');
+        expect(expression.evaluate(frameState, undefined)).toEqual(1);
+
+        expression = new Expression('rgba(255, 255, 0, 0.5)[3]');
+        expect(expression.evaluate(frameState, undefined)).toEqual(0.5);
+    });
+
+    it('evaluates color properties (["red"], ["green"], ["blue"], ["alpha"])', function() {
```

Comment:
Here and below, I see what it does, I'm just asking why since GLSL does not allow this since it does not have strings.  How would the GLSL backend handle `color[${CHANNEL}]`?

If there are compelling use cases, we can keep this.